.gitignore

```diff
@@ -5,5 +5,5 @@ __pycache__
 cache
 *.pt
 .vscode
-*.yaml
-!settings.template.yaml
\ No newline at end of file
+!settings.template.yaml
```

Comment:
The original idea was to allow multiple settings.yaml files for testing to be ignored, but I also have never really had the need for it, so I'm fine with removing it.

This line only makes sense together with a wildcard pattern. Therefore we don't need this here anymore.